### ISSUE TITLE
Probable typo in a disproof about non-antisymmetry

### ISSUE THREAD

**Issue by code-hunger** [winitzki/linear-algebra-book#1]
I believe I've found a typo in section 2.3.1, near this text:

![probable typo](https://user-images.githubusercontent.com/11772321/124310073-585b4d80-db74-11eb-9025-41078f216e78.png)

IMHO the inequality near the end should read `≠ a*(w)v` (with `w` and `v` swapped) so as to contradict equality with the first term, which _does_ equal `a*(w) v`.  
I'd be happy to submit a pull request in case I'm right and it'd be convenient.

**Comment by winitzki** [comment]
Thank you for reporting this error. I have corrected it and just pushed new source and pdf files into version-1.3-print. Please let me know if you believe this is now correct.


<img width="640" alt="Screen Shot 2021-07-03 at 5 56 17 PM" src="https://user-images.githubusercontent.com/2163489/124360073-9be9a200-dbdc-11eb-9d6b-e1f516ddb0ec.png">

**Comment by code-hunger** [comment]
Yes, thanks for the quick response, it's perfect now!

**Comment by winitzki** [comment]
Thank you. Feel free to report other errors and/or make PRs although it's probably going to be difficult for you unless you use LyX for editing. (Which I can recommend.)

**Comment by code-hunger** [comment]
Sure! And by the way, thank you for writing this great book – I used it as my first read on tensors and coordinate-free algebra and was quite satisfied!

**Comment by winitzki** [comment]
You are welcome! When I was a student, a friend of mine explained coordinate-free tensors and the wedge product to me. I was quite impressed back then, and I thought it was a great way of understanding linear algebra. Many years later, I wrote all explanations into the "linalg" book. The goal was to prove all theorems that I learned as a student, but without coordinates.

Right now I'm working on a book on functional programming. You might be interested in learning that as well. https://github.com/winitzki/sofp
